### PR TITLE
Change default EdgeCluster name

### PR DESCRIPTION
**Pull request by GraysonWu**

### CODE REVIEW

File: website/docs/r/dhcp_server_ip_pool.html.markdown [hunk 1]
```diff
@@ -14,7 +14,7 @@ Provides a resource to configure IP Pool for logical DHCP server on NSX-T manage
 
 ```hcl
 data "nsxt_edge_cluster" "edgecluster" {
-  display_name = "edgecluster1"
+  display_name = "EDGECLUSTER1"
```
Comment (annakhm) [inline website/docs/r/dhcp_server_ip_pool.html.markdown:17]: can we use the env variable here and below?

**Comment by ksamoray** [comment]
In the new topologies which I deploy with Terraform, the names are actually lowercased :)
If we want to use uppercase naming, I'll change that as well.

**Comment by salv-orlando** [comment]
These are not NSX defaults, but rather default settings for the process used to deploy NSX. As @ksamoray mentioned, we will shortly start using topologies deployed via Terraform and defaulting to lower case names.

Also, any NSX user should be able to run these tests on their own topology and have a different name for the edge cluster to use in tests.
If the above makes sense, perhaps these constants should become variables that are read from a config file.

As this might take time for the time being can use two values depending on backend NSX version?

**Comment by ksamoray** [comment]
@salv-orlando these values are just default names which are used in the testing environments, the CI sets these with environment vars and I already changed the topology to use uppercase values as the GM testing deployments use.

File: website/docs/r/dhcp_server_ip_pool.html.markdown (hunk 1 above)
Comment (salv-orlando) [reply website/docs/r/dhcp_server_ip_pool.html.markdown:17]: Let's just not make any change to the doc file. It's not relevant to this PR

File: website/docs/r/logical_dhcp_server.html.markdown [hunk 2]
```diff
@@ -14,7 +14,7 @@ Provides a resource to configure logical DHCP server on NSX-T manager
 
 ```hcl
 data "nsxt_edge_cluster" "edgecluster" {
-  display_name = "edgecluster1"
+  display_name = "EDGECLUSTER1"
```
Comment (salv-orlando) [inline website/docs/r/logical_dhcp_server.html.markdown:17]: as above

File: website/docs/r/dhcp_server_ip_pool.html.markdown (hunk 1 above)
Comment (GraysonWu) [reply website/docs/r/dhcp_server_ip_pool.html.markdown]: Changed back.

File: website/docs/r/logical_dhcp_server.html.markdown (hunk 2 above)
Comment (GraysonWu) [reply website/docs/r/logical_dhcp_server.html.markdown]: Done.

File: nsxt/utils_test.go [hunk 3]
```diff
@@ -27,7 +27,8 @@ import (
 // Default names or prefixed of NSX backend existing objects used in the acceptance tests.
 // Those defaults can be overridden using environment parameters
 const tier0RouterDefaultName string = "PLR-1 LogicalRouterTier0"
-const edgeClusterDefaultName string = "edgecluster1"
+const edgeClusterDefaultName string = "EDGECLUSTER1"
+const computeCollectionDefaultName string = "Cluster-1"
```
Comment (ksamoray) [inline nsxt/utils_test.go:31]: For local manager environment, this should be Compute-Cluster if I recall (it's the name of the VC cluster which we deploy).

Comment (ksamoray) [reply nsxt/utils_test.go:31]: BTW the value is set within the Jenkins node but it's better to have a correct default value :)

Comment (GraysonWu) [reply nsxt/utils_test.go:31]: Not sure how that env is deployed. In the test topology `Policy-L2L3-OnPrem` it's called `Cluster-1`.

**Review by ksamoray — APPROVED** [review]
